src/styles/content.scss

```diff
@@ -57,9 +60,177 @@ table.os-raise-standardtable {
   }
 }
 
+%os-raise-basictable {
+  /* border-collapse: separate paired with border-spacing: 0 allows for a border-radius
+     to be applied to the entire table (border-collapse: collapse will ignore any border
+     radius applied to the table)
+  */
+  border: 2px solid $raise-table-color;
+  border-collapse: separate;
+  border-spacing: 0;
+  border-radius: 5px;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
```

Comment:
I will address the overall px to rem font changes in this comment.

The reason I used a base of 15px is because the body font-size is .9375rem, so I interpreted that as meaning the base is 15px. I also tried to compare text using the px/rem values next to each other and it seemed like the current rem values matched the former px values, but that test was harder to make a determination because the difference is so small. Happy to change the rem values as you state throughout your comments, but wanted to note the above prior to making the change.

Reply:
That is extremely helpful! I'll keep that in mind for the future and go ahead and adjust the rem values.